spec/unit/type/service_spec.rb

```diff
@@ -273,6 +304,7 @@ def logonpassword=(value) end
 
             it "should pass validation" do
               allow(Puppet::Util::Windows::User).to receive(:localsystem?).with(predefined_local_account).and_return(false)
+              expect(Puppet::Util::Windows::User).to receive(:default_system_account?).with("#{predefined_local_account}").and_return(true)
```

Comment:
I don't think you need the interpolation here.
```suggestion
              expect(Puppet::Util::Windows::User).to receive(:default_system_account?).with(predefined_local_account).and_return(true)
```